packages/launchdarkly-adapter/src/adapter/adapter.spec.js

```diff
@@ -405,6 +404,7 @@ describe('when configuring', () => {
             expect(onFlagsStateChange).toHaveBeenCalledTimes(1);
           });
 
+          // biome-ignore lint/complexity/noExcessiveNestedTestSuites: these test suits could be reorganized
```

Comment:
biome has a limit of 5 nesting levels for describe. I think it's a good rule of thumb to follow, maybe we could reorganize these tests later on